pkg/pvc/pvc_protected_entity.go

```diff
@@ -393,6 +464,12 @@ func (this PVCProtectedEntity) getComponentPEType() string {
 	return peType
 }
 
+func (this PVCProtectedEntity) getMigrationResourcelist(ctx context.Context) (*unstructured.UnstructuredList, error) {
+	migrationGVR := schema.GroupVersionResource{Group: "cns.vmware.com", Version: "v1alpha1", Resource: "cnsvspherevolumemigrations"}
+	migrationResource := this.ppetm.dynamicClient.Resource(migrationGVR)
+	migrationResourcelist, err := migrationResource.List(ctx, metav1.ListOptions{})
+	return migrationResourcelist, err
```

Comment:
If the CNSvSphereVolumeMigration CRD is not installed, will you get an empty list and an err, or an empty list and nil?

Reply:
Will check.
For this commit, getMigrationResourcelist is called at line 302 where we checked the pv annotation already. So if we cannot list migration crd, it's an error in system and we can return error.

In general case where the system doesn't have migration crd and this dynamic client tries to access it, I'll do an experiment to check.

Reply:
Is CNSvSphereVolumeMigration CRD a non-namespaced object?

Reply:
Yes, it is cluster scope
root@k8s-control-62-1642097780:~# kubectl get  crd/cnsvspherevolumemigrations.cns.vmware.com -o=jsonpath="{.spec.scope}{'\n'}"
Cluster

Reply:
I think this should be fine.